### PR TITLE
Combining develop and dev/refactor HOPPs correctly

### PR DESCRIPTION
**Pull request by johnjasa**
I took a stab at:
- bringing `develop` into `dev/refactor` before Rob removed GreenHEART
- merging
- putting the modified HOPP back into `develop`

However, there are many files (especially examples) that don't seem explicitly GreenHEART related and might be useful for HOPP? Would love additional eyes here and comments or commits removing additional files.

### CODE REVIEW

File: hopp/simulation/resource_files/solar/27.18624_-96.9516_psmv3_60_2013.csv
Comment (kbrunik) [inline hopp/simulation/resource_files/solar/27.18624_-96.9516_psmv3_60_2013.csv:1]: Let's remove these new resource files for both solar and wind.

File: hopp/simulation/technologies/calc_power_losses.py
Comment (kbrunik) [inline hopp/simulation/technologies/calc_power_losses.py:1]: This file looks like some analysis code that is also for H2. Suggest removing.

File: hopp/simulation/technologies/layout/wind_layout.py [hunk 1]
```diff
@@ -220,12 +220,13 @@ def plot(self,
 
         turb_pos_x = self._system_model.value("wind_farm_xCoordinates")
         turb_pos_y = self._system_model.value("wind_farm_yCoordinates")
-
+        print("CALLING WIND LAYOUT PLOT")
```
Comment (kbrunik) [inline hopp/simulation/technologies/layout/wind_layout.py:223]: Maybe remove the print statement?

File: hopp/simulation/technologies/power_source.py [hunk 2]
```diff
@@ -49,7 +49,11 @@ def __init__(self, name, site: SiteInfo, system_model, financial_model):
         if isinstance(self._financial_model, Singleowner.Singleowner):
             self.initialize_financial_values()
         else:
-            self._financial_model.assign(self._system_model.export(), ignore_missing_vals=True)       # copy system parameter values having same name
+            if inspect.ismethod(getattr(self._system_model, 'export', None)):
+                self._financial_model.assign(self._system_model.export(), ignore_missing_vals=True)       # copy system parameter values having same name
+            else:
+                pass
+            # self._financial_model.assign(self._system_model.export(), ignore_missing_vals=True)       # copy system parameter values having same name
```
Comment (kbrunik) [inline hopp/simulation/technologies/power_source.py:56]: remove commented code

File: hopp/tools/hopp_tools.py
Comment (kbrunik) [inline hopp/tools/hopp_tools.py:1]: Remove

File: setup.py
Comment (kbrunik) [inline setup.py:1]: remove for `pytoml`

File: tests/hopp/hopp_for_h2_test_results.csv
Comment (kbrunik) [inline tests/hopp/hopp_for_h2_test_results.csv:1]: remove

File: hopp/add_custom_modules/custom_wind_floris.py
[the remark this answers is not in the document]
Comment (johnjasa) [reply hopp/add_custom_modules/custom_wind_floris.py:1]: removed, it's defunct with the new floris v4 addition

File: hopp/simulation/resource_files/solar/27.18624_-96.9516_psmv3_60_2013.csv
Comment (johnjasa) [reply hopp/simulation/resource_files/solar/27.18624_-96.9516_psmv3_60_2013.csv:1]: removed the new ones

File: hopp/simulation/technologies/power_source.py (hunk 2 above)
Comment (johnjasa) [reply hopp/simulation/technologies/power_source.py:56]: done

File: examples/inputs/offshore_fixed_platform.yaml
Comment (jaredthomas68) [inline examples/inputs/offshore_fixed_platform.yaml:1]: Remove `examples/inputs/offshore_fixed_platform.yaml` from HOPP

File: hopp/simulation/technologies/hydrogen/__init__.py
Comment (jaredthomas68) [inline hopp/simulation/technologies/hydrogen/__init__.py:1]: I think this whole `hopp/simulation/technologies/hydrogen/` directory should be removed from hopp

File: requirements-dev.txt
Comment (jaredthomas68) [inline requirements-dev.txt:1]: should be removed for .toml hopp

File: requirements.txt
Comment (jaredthomas68) [inline requirements.txt:1]: should be removed for `.toml` hopp

File: setup.py
Comment (jaredthomas68) [inline setup.py:1]: should be removed for `.toml` hopp

File: tests/analysis/Expected_H2_Analysis_Main.csv
Comment (jaredthomas68) [inline tests/analysis/Expected_H2_Analysis_Main.csv:1]: can remove tests/analysis/Expected_H2_Analysis_Main.csv

File: tests/hopp/hopp_for_h2_test_results.csv
Comment (jaredthomas68) [inline tests/hopp/hopp_for_h2_test_results.csv:1]: remove tests/hopp/hopp_for_h2_test_results.csv

File: tests/hopp/test_hydrogen/test_PEM_costs_Singlitico_model.py
Comment (jaredthomas68) [inline tests/hopp/test_hydrogen/test_PEM_costs_Singlitico_model.py:1]: Remove `tests/hopp/test_hydrogen/`

**Review by jaredthomas68 — CHANGES_REQUESTED** [review]
Would be good to get a thumbs up from someone else (@bayc or @elenya-grant or @kbrunik) on my comments before moving forward

**Comment by kbrunik** [comment]
> Would be good to get a thumbs up from someone else (@bayc or @elenya-grant or @kbrunik) on my comments before moving forward

For some reason the files changes show when @johnjasa opened the draft PR but I think I had also suggested all of these changes and John removed/changed all of these things. So yes long story, I'm in agreement.

**Pull request closed without merging**